cmd/yurt-iot-carrier/app/options/options.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2023 The OpenYurt Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/spf13/pflag"
+)
+
+// YurtIotCarrierOptions is the main settings for the yurt-iot-carrier
+type YurtIotCarrierOptions struct {
+	MetricsAddr          string
+	ProbeAddr            string
+	EnableLeaderElection bool
+	Nodepool             string
+	Namespace            string
+	CoreDataAddr         string
+	CoreMetadataAddr     string
+	CoreCommandAddr      string
+	EdgeSyncPeriod       uint
+}
+
+func NewYurtIotCarrierOptions() *YurtIotCarrierOptions {
+	return &YurtIotCarrierOptions{
+		MetricsAddr:          ":8080",
+		ProbeAddr:            ":8080",
+		EnableLeaderElection: false,
+		Nodepool:             "",
+		Namespace:            "default",
+		CoreDataAddr:         "edgex-core-data:59880",
+		CoreMetadataAddr:     "edgex-core-metadata:59881",
+		CoreCommandAddr:      "edgex-core-command:59882",
```

Comment:
We need to keep these parameters in sync with the configuration in PlatformAdmin in the future, could you open an issue to track this?

Reply:
Of course, I will continue to track and extend the yurt-iot-dock using the ability to customize configurations in PlatformAdmin.